### PR TITLE
fix: Copy changes

### PR DESCRIPTION
**Pull request by jaapstronks**

### CODE REVIEW

File: packages/hummingbird/src/translations/de.json [hunk 1]
```diff
@@ -30,5 +30,21 @@
...
+  "hummingbird.CampaignConfirmed.main.fundraiserDefaultTitle": "Gelungen!",
+  "hummingbird.CampaignConfirmed.main.fundraiserDefaultText": "Vielen Dank für Ihre Spende.",
+  "hummingbird.CampaignConfirmed.main.fundraiserSuccessTitle": "Hurra!",
+  "hummingbird.CampaignConfirmed.main.fundraiserSuccessText": "Danke schön. Die Zahlung war erfolgreich.",
+  "hummingbird.CampaignConfirmed.main.fundraiserPendingTitle": "Vielen Dank für Ihre Spende.",
+  "hummingbird.CampaignConfirmed.main.fundraiserPendingText": "Ihre Zahlung wird verarbeitet.",
+  "hummingbird.CampaignConfirmed.main.fundraiserFailedTitle": "Oh nein!",
+  "hummingbird.CampaignConfirmed.main.fundraiserFailedText": "Ihre Zahlung wurde anscheinend abgebrochen. Bitte versuchen Sie es erneut.",
+  "hummingbird.CampaignConfirmed.main.fundraiserShareTitle": "Bitte teilen Sie diese Spendenaktion mit Ihren Freunden und Kollegen:",
+  "hummingbird.Campaign.fundraiserCounter.message": "wurde schon gespendet"
```
Comment (markjjackson) [inline packages/hummingbird/src/translations/de.json]: Please move "hummingbird.Campaign.fundraiserCounter.message" to the same position as the en file for consistency.

File: packages/hummingbird/src/translations/nl.json [hunk 2]
```diff
@@ -56,5 +56,6 @@
   "hummingbird.Footer.madeBy.emoji": "liefde",
   "hummingbird.Footer.madeBy.message": "Met {love} gemaakt door Bolster",
   "hummingbird.Footer.poweredBy.message": "Mogelijk dankzij Wings",
-  "hummingbird.Article.skipToContent.tooltip": "ga naar de inhoud"
+  "hummingbird.Article.skipToContent.tooltip": "ga naar de inhoud",
+  "hummingbird.Campaign.fundraiserCounter.message": "is al gedoneerd"
```
Comment (markjjackson) [inline packages/hummingbird/src/translations/nl.json]: "hummingbird.Campaign.fundraiserCounter.message" already exists in the object. Remove this line and edit the existing one.

**Comment by jaapstronks** [comment]
@markjjackson ok, fixed it

**Comment by markjjackson** [comment]
@jaapstronks there are some fields that need translations

**Comment by jaapstronks** [comment]
@markjjackson I think it is fixed now?

**Pull request merged**